configure

```diff
@@ -2974,9 +2982,14 @@ POSTGRES_BUILDDIR="$POSTGRES_BUILDDIR"
 
 ac_config_files="$ac_config_files Makefile.global"
 
-ac_config_headers="$ac_config_headers src/include/citus_config.h"
+ac_config_headers="$ac_config_headers src/include/citus_config.h src/include/citus_version.h"
+
 
 
+cat >>confdefs.h <<_ACEOF
+#define CITUS_BINARY_VERSION "$citus_binary_version"
```

Comment:
Nope. Thanks to [this](https://github.com/citusdata/citus/pull/1277/files#diff-2dee16a9eca5258df0e9172404e450efR48), when citus.control file changed, running just make will trigger configure automatically.